core/server/app.go

```diff
@@ -111,6 +113,46 @@ func (as *appServer) ListApps(ctx context.Context, msg *pb.ListAppRequest) (*pb.
 	}, nil
 }
 
+func (as *appServer) ListFluxRuntimeObjects(ctx context.Context, msg *pb.ListFluxRuntimeObjectsReq) (*pb.ListFluxRuntimeObjectsRes, error) {
+	k8s, err := as.k8s.Client(ctx)
+	if err != nil {
+		return nil, doClientError(err)
+	}
+
+	list := &appsv1.DeploymentList{}
+
+	if err := k8s.List(ctx, list, client.InNamespace(msg.Namespace)); err != nil {
```

Comment:
I think we should avoid making assumptions about flux namespace (https://github.com/weaveworks/weave-gitops/pull/1451/files#diff-35acf2adc3a7b1021fb41d3473b47e030d3708ba2e8ff7a4b76c13ee80b4076cR42) and grab it by using something like: https://github.com/weaveworks/weave-gitops/blob/v2/pkg/kube/kubehttp.go#L496

I'm not sure we want to keep that interface but the actual method is useful.

Reply:
@luizbafilho Since we allow the client to specify the `namespace`, can I fetch the correct namespace in a different call (in a future PR)?

Reply:
Yeah, it would be a good follow up issue

Reply:
https://github.com/weaveworks/weave-gitops/issues/1465